frontend/src/components/common/Address.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import * as Strings from "@/constants/strings";
+import { Icon } from "antd";
+import { formatPostalCode } from "@/utils/helpers";
+
+const propTypes = {
+  address: PropTypes.objectOf(PropTypes.string).isRequired,
+};
+
+/**
+ * @constant Address- reusable component that formats any given address object
+ */
+
+export class Address extends Component {
+  addressContainsTruthyValues = (object) =>
+    // address_type_code is always "CAD" && if "suit_no" exists but nothing else, this field isn't helpful
+    Object.keys(object)
+      .filter((key) => key !== "address_type_code" && key !== "suite_no")
+      .some((key) => object[key]);
+
+  formatRowContent = (itemArr) =>
+    itemArr
+      .filter((item) => item)
+      .map((item, i) => (
+        <span key={item}>{i !== itemArr.length - 1 ? `${item}${" "}` : item}</span>
```

Comment:
Did Prettier automatically format the whitespace like that? It seems like the `${" "}` should be able to be replaced with normal whitespace

Reply:
yes it did, and fixed!